wicket-core/src/main/java/org/apache/wicket/markup/html/image/Image.java

```diff
@@ -316,8 +316,9 @@ public void setXValues(String... values)
 		if (xValues == null)
 		{
 			xValues = new ArrayList<>();
+		}else{			
```

Comment:
Empty space after `{` and `else` should be on new line. Is Eclipse formatter broken or what ? ;-)